components/Header/Navigation.tsx

```diff
@@ -5,14 +5,14 @@ import { Button } from "@/components/ui/button";
 
 export default function Navigation() {
   return (
-    <nav className="hidden md:block">
-      <ul className="mr-[22px] grid list-none grid-flow-col place-items-center gap-9">
+    <nav className="hidden md:block ">
+      <ul className="mr-[22px] grid list-none grid-flow-col place-items-center gap-9 ">
         {navItems.map((item, i) => (
           <li key={i}>
             <Button variant={"link"} className="p-0 text-white" asChild>
               <Link
                 href={item.href}
-                className="text-right text-sm font-semibold leading-snug text-indigo-100 hover:text-slate-50 hover:underline lg:leading-[25px]"
+                className="text-right font-semibold leading-snug text-[sm] text-indigo-100 hover:text-slate-50 hover:underline lg:leading-[25px]"
```

Comment:
#What is `[sm]` ? does this need to be defined somewhere? How is this being read?

Reply:
@cbtsao47 its tailswind default 14px font size